style.css

```diff
@@ -282,3 +281,55 @@ h1 span {
   border-bottom: 8px;
   background-color: #ebf0ee;
 }
+
+.contact {
+  background: no-repeat 100% 100% url(./images/illustration-contact-form.png);
+  padding: 64px 24px 64px 24px;
+}
+
+.contact h3 {
+  height: 363px;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+input {
+  height: 48px;
+  width: 320px;
+  border-radius: 4px;
+  margin-top: 20px;
+  padding: 15px;
+  border: 1px solid #d0d9d4;
+}
+
+textarea {
+  height: 114px;
+  width: 320px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 15px;
+  color: #091e42;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+}
+
+.btn-green {
+  background-color: #36b37f;
+  height: 48px;
+  width: 129px;
+  border-radius: 4px;
+  padding: 12px;
+  display: flex;
+  margin: auto;
+  color: white;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+}
```

Comment:
- [ ] Please, consider the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.